config/admins.yml

```diff
@@ -0,0 +1,20 @@
+default: &default
+  emails:
+    - uche.chilaka@gmail.com
+    - donnadieu.86@gmail.com
+  ip_addresses:
+    - 23.244.86.78 # Uche's home remote IP
+
+development:
+  <<: *default
+  ip_addresses:
+    - 127.0.0.1
+    - 23.244.86.78 # Uche's home remote IP
```

Comment:
LMK your home IP address so I can add it to this list soon. I might look into implementing basic authentication for admin user accounts that will use a password field to authenticate against Flipper since we might want to work with feature flags away from a home office - a VPN is another way we can solve that problem but IDK how much it would cost to set one up with a dedicated IP 

Either way, this gives us access with network constraints (vs no access right now LOL)